slam-libraries/viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -131,6 +135,7 @@ class SLAMServiceImpl final : public SLAMService::Service {
     double occupied_space_weight = 20.0;
     double translation_weight = 10.0;
     double rotation_weight = 1.0;
+    int trajectory_id = 0;
```

Comment:
Let's not make `trajectory_id` a "global" slam service variable for now. I am not sure if we won't be handling multiple trajectory ids in other slam service functions in the future, so we should keep this variable local to functions as long as we can do so.